src/components/tutorial-markdown-render/index.tsx

```diff
@@ -67,14 +67,14 @@ const TutorialMarkdownRender = (props: Props) => {
       </Head>
       <PageHeader
         title={intl.formatMessage({
-          id: 'landing_page_tutorials_page.title',
+          id: 'documentation_tutorials.title',
         })}
         description={intl.formatMessage({
-          id: 'tutorials_page.description',
+          id: 'documentation_tutorials.description',
         })}
         imageUrl={startHereImage}
         imageAlt={intl.formatMessage({
-          id: 'landing_page_tutorials_page.title',
+          id: 'documentation_tutorials.title',
```

Comment:
@PedroAntunesCosta I've made these changes to strings here since there is no longer `landing_page_tutorials_page.title`. However, I've noticed that this page is related to each tutorial category, right? 
I've screenshotted the following page just so I can have a prior context of this page layout:
- Does the tutorial page category's purpose have the "Main header"  repeating the tutorial title and description?
- I've noticed the page also has a breadcrumb. What do you think about instead of having this "Main header" we have only the breadcrumb as `Tutorials > {category-name}`, in the screenshot case, `Tutorials > Apps`.

![Screenshot_407](https://github.com/user-attachments/assets/eff19bd5-4b21-4a3b-8da5-8a7a171bea72)

Reply:
Good questions, @mariana-caetano ! But I don't think we can act on it. At least not in this precise way.

The tutorials category page is just a tutorial based in an [automatically generated .md file](https://github.com/vtexdocs/help-center-content/blob/main/docs/en/tutorials/Apps/category-apps.md) listing subcategories and article links.

So changing these pages' layout means changing all tutorials layout.

![image](https://github.com/user-attachments/assets/3103fd47-386c-42cb-8774-3c10a8b05edc)

I don't know the design rationale behind these choices, but maybe it would be worth to tackle these points in a future design sprint.

Reply:
By these points I mean:

1. Why is the section name not included in the breadcrumbs?
2. Why do tutorials have that big header, while other content types don't (e.g., FAQ 👇)?

![image](https://github.com/user-attachments/assets/e37cbc87-cff4-4c3e-8e70-9b995236a804)

Reply:
Thanks @PedroAntunesCosta!

I will open an issue for these points so we can have a look later on.